### PR TITLE
Subsequence of string using bitmasking

### LINKED ISSUE #1722: Subsequence of string using bitmasking
I would like to contribute to the code in C++
Subsequence of a string are always the left shift of length of the string.
if the string is "abc" the number of subsequences will be 8.
a ab abc b bc c ac "" . 1<<n (where n is the length of the string) 
Please assign me this issue as participant of GSSOC 20.
@SakshiShreya  @singh-shreya6  @asha15  @raghav-dalmia

### PR DESCRIPTION
**Pull request by ashishnagpal2498**
Fixes #1722 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Languages Used:
- C++
- 

#### Files Added:
- Subsequence_bitmasking 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Subsequence_Bitmasking/subsequence_bitmasking.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include<string>
+using namespace std;
+void subsequence_bitmasking(string s)
+{
+    //Calculate length of string 
+    int len = s.length();
+
+    //Number of subsequence are always 1>>length
+    
+    for(int i=0;i<(1<<len);i++)
```
Comment (jainaman224) [inline Subsequence_Bitmasking/subsequence_bitmasking.cpp]: Please add spaces around operator like `+`, `=` etc and space after operator like `;`.

File: Subsequence_Bitmasking/subsequence_bitmasking.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+}
+int main()
+{
+    string str;
+    cout<<"Enter string"<<endl;
+    cin>>str;
+    subsequence_bitmasking(str);
+    return 0;
+}
```
Comment (jainaman224) [inline Subsequence_Bitmasking/subsequence_bitmasking.cpp]: Add sample input output.
add a new line at the end of file.

Comment (ashishnagpal2498) [reply Subsequence_Bitmasking/subsequence_bitmasking.cpp]: Done

File: Subsequence_Bitmasking/subsequence_bitmasking.cpp [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                ans += s[m];
+            }
+            m++;
+            j = j >> 1;
+        }
+        cout << ans << " ";
+    }
+}
+int main()
```
Comment (jainaman224) [inline Subsequence_Bitmasking/subsequence_bitmasking.cpp]: Add a new line above functions.

File: Subsequence_Bitmasking/subsequence_bitmasking.cpp [hunk 4]
```diff
@@ -0,0 +1,48 @@
+//Subsequence of a String using bitmasking
+#include<iostream>
+#include<string>
+using namespace std;
+void subsequence_bitmasking(string s)
+{
+    //Calculate length of string
```
Comment (jainaman224) [inline Subsequence_Bitmasking/subsequence_bitmasking.cpp]: Add a space after `//`.

**Pull request merged**